src/get-started/wordcount-example.md

```diff
@@ -773,12 +773,21 @@ To view the full code in Java, see
 
 > **Note:** WindowedWordCount is not yet available for the Python SDK.
 
-### Unbounded and bounded pipeline input modes
+### Unbounded and bounded data sets
 
 Beam allows you to create a single pipeline that can handle both bounded and
 unbounded types of input. If your input has a fixed number of elements, it's
-considered a 'bounded' data set. If your input is continuously updating, then
-it's considered 'unbounded' and you must use a runner that supports streaming.
+considered a bounded data set. For bounded data sets, the question you ask is
```

Comment:
"If your input has a fixed number of elements, it is a bounded data set, and all of the data can be processed together. If input data continuously arrives (such as...), it is an unbounded data set. An unbounded data set will only be complete up to a certain point, and must be processed with..."

(roughly)